api/controllers/case.js

```diff
@@ -235,11 +235,9 @@ async function postCaseUpdateHttp(req, res) {
   if (!er.hasErrors()) {
     if (updatedText) {
       await db.tx("update-case", t => {
-        return t.batch([
-          t.none(INSERT_AUTHOR, author),
-          t.none(INSERT_LOCALIZED_TEXT, updatedText),
-          t.none(UPDATE_CASE, updatedCase)
-        ]);
+        t.none(INSERT_AUTHOR, author),
+        t.none(INSERT_LOCALIZED_TEXT, updatedText),
+        t.none(UPDATE_CASE, updatedCase)
       });
```

Comment:
@pan this doesn't look correct to me. looking at the original PR these lines should be: 
https://github.com/participedia/api/pull/773/files#diff-07780e70b8cc879c37de679a75e0e6bbR235

```
await db.tx("update-organization", async t => {
  await t.none(INSERT_AUTHOR, author);
  await t.none(INSERT_LOCALIZED_TEXT, updatedText);
  await t.none(UPDATE_ORGANIZATION, updatedOrganization);
});
```



@pan i'm curious, why didn't you merge the original PR's?
https://github.com/participedia/api/pull/775
https://github.com/participedia/api/pull/774
https://github.com/participedia/api/pull/773

Reply:
> @pan this doesn't look correct to me. looking at the original PR these lines should be:
> https://github.com/participedia/api/pull/773/files#diff-07780e70b8cc879c37de679a75e0e6bbR235
> 
> ```
> await db.tx("update-organization", async t => {
>   await t.none(INSERT_AUTHOR, author);
>   await t.none(INSERT_LOCALIZED_TEXT, updatedText);
>   await t.none(UPDATE_ORGANIZATION, updatedOrganization);
> });
> ```
> 
> @pan i'm curious, why didn't you merge the original PR's?
> #775
> #774
> #773

Now that looks right. The code I mentioned in the beginning looks very odd.

Reply:
@ascott I didn't just merge the 3 original PRs because when I checked they were already merged before I started this task https://github.com/participedia/api/issues/809. So I assume I need to make changes to how we are using db.tx and t.batch from the pg-promise lib.

Reply:
@paninee hmm, not sure how/why they appeared to be already merged since the PRs are still open and not merged. I would suggest pulling these 3 PRs (#775, #774, #773) into a new branch, resolving conflicts and make sure the tests are passing, and then opening a new PR. Please tag me and I will take a look before merging into master. Thanks!

Reply:
Will do. Thanks @ascott